src/big_mood_detector/infrastructure/parsers/json/fitbit_sleep_parser.py

```
@@ -0,0 +1,38 @@
+                        start_date=start,
+                        end_date=end,
+                        state=SleepState.ASLEEP,
+                    )
+                )
+            except (ValueError, TypeError) as error:
+                logger.debug("fitbit_sleep_parse_error", error=str(error))
+        return records
+
+    def _parse_datetime(self, value: str | None) -> datetime:
+        if not value:
+            raise ValueError("Missing Fitbit datetime")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)